301-metadata-repositories.md

```diff
@@ -0,0 +1,307 @@
+---
+title: CNAB Security: Metadata repositories
+weight: 301
+---
+
+# Cloud Native Application Bundles Security (CNAB-Sec): Metadata repositories 1.0 WD
+
+* [Metadata repositories](#metadata-repositories)
+  * [Example for multiple projects](#example-for-multiple-projects)
+    * [Administrators](#administrators)
+    * [Developers](#developers)
+    * [Automation](#automation)
+    * [Security analysis](#security-analysis)
+* [References](#references)
+
+This document is a _normative_ part of [CNAB Security](300-CNAB-security.md).
+
+The keywords MUST, MUST NOT, REQUIRED, SHALL, SHALL NOT, SHOULD, SHOULD NOT, RECOMMENDED, MAY, and OPTIONAL in this document are to be interpreted as described in [RFC 2119](https://tools.ietf.org/html/rfc2119). The use of these keywords in this document are statements about how a CNAB implementation may fulfill the CNAB Security specification _only_.
+
+## Metadata repositories
+
+A _metadata repository_ is a service that hosts TUF and / or in-toto metadata about bundles and / or images. The repository is conceptually distinct from a [CNAB registry](200-CNAB-registries.md), which hosts bundles and / or images themselves.
+
+How TUF and in-toto metadata should be designed for a metadata repository depends on which purpose it serves. The repository may be _private_ for internal consumption, or _public_. Authentication to private metadata repositories are out of the scope of this document.
+
+### Example for multiple projects
+
+This subsection discusses how an organization (e.g., `example.com/example-org/*`) MAY setup a metadata repository that hosts metadata about bundles and / or images developed and / or maintained by different _projects_, groups of developers. This repository MAY be private or public.
+
+![Figure 1: The suggested metadata repository for bundles and / or images developed and / or maintained by different projects, or groups of developers](img/example-metadata-repository.png)
+
+Figure 1 illustrates this suggested metadata repository, which we discuss using a top-down, outside-in approach.
+
+#### Administrators
+
+The four-top level TUF `root`, `timestamp`, `snapshot`, and `targets` roles SHOULD be controlled by the repository _administrators_, who develop and maintain the repository's hardware and software.
```

Comment:
I would link all TUF and in-toto terms to the relevant sections in the respective specifications.